Derikka/CavityTest_code.py

```diff
@@ -0,0 +1,68 @@
+from pydm import Display
+from PyQt5 import QtGui, QtCore, QtWidgets
+import epics
+from epics import caget, caput
+
+class CavityTest(Display):
+
+	def ui_filename(self):
+		return 'CavityTest.ui'	
+
+	def __init__(self, parent=None, args=None):
+		super(CavityTest, self).__init__(parent=parent, args=args)
+
+		# Parameters
+		self.PV = ''									# current PV
+		self.Test_PV = 'SIOC:SYS0:ML07:AO010'			# PV string
+		self.value = caget(self.Test_PV)				
+
+		# Button Connections (send signal and then call a function)
+		self.ui.Qt_pvPushButton.clicked.connect(lambda:self.updatePVOutput())				
+
+		self.ui.Qt_radioButton0.PV = self.Test_PV		
```

Comment:
Also I'd name these something more descriptive! Like, what _is_ `Qt_radioButton0`? I haven't looked at it yet, but just as an example, it could be something like `displaySizeRadioButton`